examples/3d/wireframe.rs

```diff
@@ -25,7 +25,12 @@ fn setup(
     mut materials: ResMut<Assets<StandardMaterial>>,
 ) {
     // To draw the wireframe on all entities, set this to 'true'
-    wireframe_config.global = false;
+    wireframe_config.debug_wireframes = false;
```

Comment:
I'm not sure why the original example was like that, but I feel like global configs like that should be done when inserting the resource. This is how it's done in most other example even this example uses that pattern for other configurations.

Reply:
Also, I'm not sure how to comment on the line since it hasn't been touched by the PR, but is the `WgpuFeatures::POLYGON_MODE_LINE` necessary?

Reply:
Yeah, I know it's used, it's the part where it's declared in the example code that I'm not sure is necessary, at least, it works for me without declaring it.

Reply:
I think it might be just Vulkan that requires it to be enabled manually, at least after a cursory glance at wgpu HAL (see [here](https://github.com/gfx-rs/wgpu/blob/e54a36ee78693aca0120458bcc2c03262a715f1e/wgpu-hal/src/vulkan/adapter.rs#L120)). What backend are you testing on?

Reply:
I'm using the default that wgpu picks which happens to be vulkan in my case. I never specified the WgpuFeatures but I can still clearly see the wireframes

Reply:
Nvidia driver? I've had it enable features without complaining in the past, but the Khronos validation layers should catch the invalid usage (assuming that there isn't something else in Bevy that is enabling the feature...)

Reply:
I'm testing on a windows 10 machine and this is the debug info from bevy
```log
INFO bevy_render::renderer: AdapterInfo { name: "NVIDIA GeForce RTX 2070", vendor: 4318, device: 7938, device_type: DiscreteGpu, backend: Vulkan }
```

Reply:
I don't get any validation error

Reply:
I guess it might be platform dependent. In that case, leave it there, but I would still recommend moving the plugin configuration inside an init resrouce

Reply:
Hmm, I just tried Windows+Vulkan and it is indeed not printing any validation error (even if the feature is disabled)

Reply:
At the same time, I guess it won't hurt to keep the flag enabled? I'm worried this might just be ICDs silently ignoring the flag, but I can't explain why validation isn't complaining 🤔

Reply:
I moved the config to a `insert_resource` call in any case